sources/operator.1.11.0.pxc-operator.json

```diff
@@ -0,0 +1,246 @@
+        },
+        "pmm": {
+          "2.28.0": {
+            "image_path": "percona/pmm-client:2.28.0",
+            "image_hash": "87b72578691b168f821d71f9a1120c484def570dcdac47767041a5ad0a8be90c",
+            "status": "recommended",
+            "critical": false
+          }
+        },
+        "proxysql": {
+          "2.3.2-1.1": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.11.0-proxysql",
+            "image_hash": "5ca4d85015d0501492462efedc22d4e860423cdf2fa07f1eafbcc11a60bb0844",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.0.18-2": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-proxysql-8.0.25",
+            "image_hash": "b84701c47a11c6f5ca46481f25f1b6086c0a30014d05584c7987f1d42a17b584",
+            "status": "available",
+            "critical": false
+          },
+          "2.0.18": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-proxysql",
+            "image_hash": "f109a62eb316732d59dd80ed0e013fc9594cbae601586b94023b8c068f7ced7b",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "haproxy": {
+          "2.4.15": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.11.0-haproxy",
+            "image_hash": "ebeb5b882394fd2a2c5ab302ecb410745a34bbe472a0846efb17ed8f234cf933",
+            "status": "recommended",
+            "critical": false
+          },
+          "2.3.15": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-haproxy-8.0.25",
+            "image_hash": "62479be2a21192a3215f03d3f9541decd5ef1737741245ac33ee439915a15128",
+            "status": "available",
+            "critical": false
+          },
+          "2.3.14": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-haproxy",
+            "image_hash": "2f06ac4a0f39b2c0253421c3d024291d5ba19d41e35e633ff6ddcf4ba67fd51a",
+            "status": "available",
+            "critical": false
+          }
+        },
+        "backup": {
+          "8.0.27": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.11.0-pxc8.0-backup",
+            "image_hash": "37a087035f74c26c67e44dd437f6128b5af419902cd416b17f72f82eaaaf2dad",
+            "status": "recommended",
+            "critical": false
+          },
+          "8.0.25": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc8.0.25-backup",
+            "image_hash": "c3991f0959a3b4114d7ff629d9d3cdf0dc200c58443ca8ebb1446d8b1cbe416d",
+            "status": "available",
+            "critical": false
+          },
+          "8.0.23": {
+            "image_path": "percona/percona-xtradb-cluster-operator:1.10.0-pxc8.0-backup",
+            "image_hash": "6ab8efb3804d1e519e49ee10eb46b428a837cfdcee222cc5ae2089cc1dc02a6d",
+            "status": "available",
+            "critical": false
+          },
```

Comment:
I think these need to be removed. Only one backup image should exists for major version unless we add more after the release.